CONTRIBUTING.np.md

```diff
@@ -0,0 +1,156 @@
+# Contributing
+
+🎌
+[Čeština][CZ],
+[Deutsch][DE],
+**English**,
+[Español][ES],
+[Français][FR],
+[Italiano][IT],
+[Kurdi][KU],
+[Lietuvių][LT],
+[Nederlands][NL],
+[Norsk][NO],
+[Polski][PL],
+[Português][PT_BR],
+[Türkçe][TR],
+[Ελληνικά][GR],
+[العربية][AR],
+[日本語][JA],
+[正體中文][ZH_TW],
+[简体中文][ZH_CN],
+[한국어][KO_KR]
+[नेपाली][NP]
```

Comment:
Please can this be alphabetically sorted (Nepali between AR and JA links) and replace the link with bold text (`**नेपाली**`)